.gitignore

```
@@ -11,6 +11,7 @@ public
 node_modules/
 .tmuxp.*
 .idea
```

.tmp should go in here as well

```suggestion
.idea
.tmp
```

---

Actually not needed, I just noticed that you're putting a .gitignore in the .tmp folder, which is a great idea.